methods.tex

```diff
@@ -360,61 +380,38 @@ \subsection{Material Flow}
 
 \section{Scenario Specifications}
 
-The scenario specifications  are
-listed in tables \ref{tab:gen}, \ref{tab:sim_eu}, and \ref{tab:sim_france}.
+The scenario specifications  are listed in table \ref{tab:gen}.
 The reprocessing and \gls{MOX} fabrication capacity in France
 prior to 2020 is modeled after the 
-French La Hague and MELOX site \cite{schneider_spent_2008, hugelmann_melox_1999}.
+French La Hague and MELOX sites \cite{schneider_spent_2008, hugelmann_melox_1999}.
 
 
 \begin{table}[h]
     \centering
+    \caption {Simulation Specifications}
     \begin{tabularx}{\textwidth}{bb}
         \hline
         \textbf{Specification} &\textbf{ Value} \\
         \hline
-        Simulation Time & 1970-2160 \\ 
-        Reprocessed Uranium Usage &  None. Stockpile reprocessed U \\
-        Storage Residence Time & 36 months \\
-        \gls{SFR} available year & 2040 \\
+        Simulation Starts & 1970 \\
+        Simulation Ends & 2160 \\ 
         Production of \gls{ASTRID} fuel begins & 2020 \\
+        Year \glspl{SFR} become available & 2040 \\
```

Comment:
Delete "Year". We know 2040 is a year, particularly if you have a third column for units.